### PR TITLE
Update modules and images

### PR DESCRIPTION
**Pull request by Michkov**
- Update images to RHEL9 and golang 1.19
- Update application-service and application-api modules

### CODE REVIEW

File: pkg/pipeline-selector/pipeline_selector_test.go [hunk 1]
```diff
@@ -450,7 +450,32 @@ func TestGetPipelineSelectionParametersForComponent(t *testing.T) {
                 components:
                   - name: outerloop-deploy
                     kubernetes:
-                        uri: outerloop-deploy.yaml
```
Comment (mkosiarc) [inline pkg/pipeline-selector/pipeline_selector_test.go:453]: What is the reason behind this change?

Comment (Michkov) [reply pkg/pipeline-selector/pipeline_selector_test.go:453]: the devfile validation is more strict now and trying to resolve the file which does not exist.

**Review by mkosiarc — APPROVED** [review]

**Review by tkdchen — APPROVED** [review]
